pkgs/development/python-modules/pycdio/default.nix

```diff
@@ -1,54 +1,57 @@
 {
   lib,
-  stdenv,
   buildPythonPackage,
-  fetchPypi,
+  fetchFromGitHub,
   setuptools,
-  nose,
-  pkgs,
+  pkg-config,
+  swig,
+  libcdio,
+  libiconv,
+  pytestCheckHook,
 }:
 
 buildPythonPackage rec {
   pname = "pycdio";
-  version = "2.1.1";
-  format = "setuptools";
+  version = "2.1.2dev.0";
```

Comment:
The Nixpkgs standard here is `2.1.1-unstable-2024-02-26`. I asked upstream and apparently they don’t plan to cut an actual release for some months yet, so i don’t think this counts as an actual release version number.